tests/compass-runtime-agent/README.md

```diff
@@ -0,0 +1,14 @@
+# Compass Runtime Agent Tests
+
+## Overview
+
+This project contains the acceptance tests that you can run as the Kyma Compass Runtime Agent testing process.
+The tests are written in Go. Run them as standard Go tests.
+
+## Usage
+
+This section provides information on building and versioning of the Docker image, as well as configuring the Kyma.
```

Comment:
That is not true, this document tells me nothing. Please provide more information.

Reply:
Done!